src/aead/gcm.rs

```diff
@@ -37,6 +37,7 @@ impl Key {
                 }
             }
 
+            #[cfg(not(target_arch = "mips64"))]
```

Comment:
I believe the new version of this file in master will do the right thing for you automatically.